lib/testing/src/lib/process-services-cloud/actions/application.ts

```diff
@@ -59,7 +59,7 @@ export class Application {
                 Logger.error(`[Application] Undeploy application ${applicationName} failed with error: ${error.message}`);
             }
         };
-        return ApiUtil.waitForApi(apiCall, isApplicationUndeployed, 10, 3000);
+        return ApiUtil.waitForApi(apiCall, isApplicationUndeployed, 10, 15000);
```

Comment:
why 15000 and not 16000

Reply:
Sorry, closing this pr. I made changes in undeploy method instead of delete method. I am increasing the delay to 15 seconds before deleting descriptor.